### PR TITLE
Dev crypto test

### PR DESCRIPTION
**Pull request by echatav**

### CODE REVIEW

File: scripts/install_secp.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+cd crypto
+git clone https://github.com/bitcoin-core/secp256k1
+cd secp256k1
+./autogen.sh
+./configure --enable-jni --enable-experimental --enable-module-schnorr --enable-module-ecdh --prefix=$PWD/.tmp
+make install
```
Comment (kirkwood) [inline scripts/install_secp.sh]: `make` is probably sufficient.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -22,6 +22,8 @@ object Dependencies {
   val lmdb                 = "org.lmdbjava"           % "lmdbjava"         % "0.6.0"
   val shapeless            = "com.chuusai"            %% "shapeless"       % "2.3.2"
   val scalaCheck           = "org.scalacheck"         %% "scalacheck"      % "1.13.4" % "test"
+  val guav                 = "com.google.guava"       % "guava"            % "16.0"
```
Comment (kirkwood) [inline project/Dependencies.scala]: Not sure I understood this change?

File: scripts/install_secp.sh (hunk 1 above)
Comment (echatav) [reply scripts/install_secp.sh]: oh, right, with `make` without `install` we wouldn't need the `--prefix`, right?

File: project/Dependencies.scala (hunk 2 above)
Comment (echatav) [reply project/Dependencies.scala]: It was a fix for this error:
https://travis-ci.org/rchain/rchain/jobs/343519746#L979

**Pull request merged**